manifest_staging/charts/secrets-store-csi-driver/templates/role.yaml

```diff
@@ -50,14 +50,4 @@ rules:
   - get
   - patch
   - update
-{{- if .Values.rbac.pspEnabled }}
```

Comment:
I think `make manifests` is removing the psp section. We'll need to add this manually to both `charts/` and `manifest_staging/charts` and then add it as part of kustomize. WDYT?